extension.json

```diff
@@ -8,6 +8,21 @@
 	],
 	"descriptionmsg": "bs-distributionconnector-extension-description",
 	"type": "bluespice",
+	"attributes": {
+		"BlueSpiceFoundation": {
+			"PermissionRegistry": {
+				"editwidgets": {
+					"type": "namespace",
+					"roles": [
+						"admin"
+					]
+				}
+			}
+		}
+	},
+	"AvailableRights": [
```

Comment:
AvailableRights does not need to be set here. The widget extension is responsible for registering its rights to mediawiki